AdyenComponents/BACS Direct Debit/Factories/BACSDirectDebitItemsFactory.swift

```diff
@@ -0,0 +1,170 @@
+//
+// Copyright (c) 2021 Adyen N.V.
+//
+// This file is open source and available under the MIT license. See the LICENSE file for more info.
+//
+
+import Adyen
+import Foundation
+
+internal protocol BACSDirectDebitItemsFactoryProtocol {
```

Comment:
What is the intention of this protocol exactly? It does not seem like a reusable interface or a type constraint so curious what it accomplishes